.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: node_js
-
 node_js:
-  - "11"
+  - "lts/*"
```

Comment:
Note that current LTS version is now 16, which we haven't really tested with yet

Reply:
yeah, but remoteresource and remoteresources3 both use `lts/*` so i figured it was ok. at least it'll be consistent